### PR TITLE
Add nodeSelector, tolerations, affinity configuration to helm chart

### PR DESCRIPTION
**Pull request by YunSangJun**
I added nodeSelector, tolerations, affinity configuration to helm chart to assign pods to nodes.

### CODE REVIEW

**Comment by YunSangJun** [comment]
@vito @jessehu @edevil 
What does mean the following error message from `continuous-integration/vic/pr`
```
+ echo YunSangJun
YunSangJun
+ echo $SKIP_CHECK_MEMBERSHIP

+ if $SKIP_CHECK_MEMBERSHIP == true; then echo 'check-org-membership step skipped'; else /bin/bash -c '[[ ! $(curl --silent "https://api.github.com/orgs/vmware/members/YunSangJun?access_token=$GITHUB_AUTOMATION_API_KEY") ]]'; fi
/bin/sh: 23: ==: not found
```
And one more, In `coverage/coveralls` test, I do not know what is the problem. I hope you give me a hint.

**Comment by jessehu** [comment]
@YunSangJun No worry. That CI does not allow PR from non-VMware users. I triggered a new one here https://ci-vic.vmware.com/vmware/harbor/4481 . Please wait for a while.

**Comment by jessehu** [comment]
Could you describe the purpose of adding these attributes? Does your project require them ?

**Comment by YunSangJun** [comment]
@jessehu Thank you. 
Our team wants to run Harbor only at certain nodes. Helm 2.8.0+ also generate these configuration by default. The Kubernetes Official Chart Guide also recommends including this attribute.

File: contrib/helm/harbor/templates/adminserver/adminserver-ss.yaml [hunk 1]
```diff
@@ -56,6 +56,18 @@ spec:
           items:
           - key: secretKey
             path: key
+    {{- with .Values.adminserver.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
```
Comment (jessehu) [inline contrib/helm/harbor/templates/adminserver/adminserver-ss.yaml]: why not using `toYaml .Values.adminserver.nodeSelector | indent 8` but `with .Values.adminserver.nodeSelector` ?

**Comment by jessehu** [comment]
Make sense. Thanks. And could you let us know which company you work for if it's convenient ? We'd like to update the internal list of Harbor customers :)

File: contrib/helm/harbor/templates/adminserver/adminserver-ss.yaml (hunk 1 above)
Comment (YunSangJun) [reply contrib/helm/harbor/templates/adminserver/adminserver-ss.yaml]: I followed the template from `helm create`.

**Comment by YunSangJun** [comment]
I work at SK Holdings C&C. We provide IT outsourcing services to our affiliates.

**Pull request merged**